### ISSUE TITLE
Disabling Squiz sniff

### ISSUE THREAD

**Issue by josephzidell** [squizlabs/PHP_CodeSniffer#1796]
```php
<?php
// phpcs:disable Squiz.PHP.ForbiddenFunctions
print_r();
// phpcs:enable
```

This gives a warning, but shouldn't.

**Comment by gsherwood** [comment]
The error code for that message is `Squiz.PHP.DiscouragedFunctions.Discouraged` so you need to disable `Squiz.PHP.DiscouragedFunctions`

**Comment by josephzidell** [comment]
Got it. This works.
